### DISCUSSION TITLE
Help porting from Aniseed to nfnl

### DISCUSSION THREAD

**Discussion by lc9er** [Olical/nfnl#20] [Q&A] [answered]
I'll start by stating my problem is probably due to my lack of understanding the nvim api and/or Aniseed and nfnl.

I have a small plugin that I'm porting from Aniseed and I seem to be running into trouble with this last section:

Aniseed version

```
(defn init []
  (nvim.ex.command_
    :-nargs=* :-complete=command :Bran :lua
    "require('bran.main')['bran-bran'](<f-args>)"))

(init [])
```

Is there an equivalent to 'ex.command_' in nfnl?

**Comment by Olical** [comment, marked as answer]
So `nvim.*` was actually https://github.com/norcalli/nvim.lua which is now merged into Neovim core (has been for ages actually). I used to keep people using `nvim.lua` because it had some extras that didn't get merged but now I don't think it's wroth it and I think it's worth using the built ins since they're so good now.

The "native" way to call an ex command now is https://neovim.io/doc/user/lua.html#vim.cmd()

**Comment by lc9er** [comment]
That was exactly the pointer I needed. Thank you for your help!
